### DISCUSSION TITLE
Error: connect ECONNREFUSED Postman

### DISCUSSION THREAD

**Discussion by Takepaty** [balta-io/2811#25] [Q&A] [answered]
Boa tarde! 
Estou fazendo o curso Fundamentos do ASP.NET.
Baixei o app postman e colei o endereço localhost no get e cliquei no enter.
Apareceu o seguinte erro 

![image](https://user-images.githubusercontent.com/102318922/175128834-01c26273-8bfd-45b6-a25b-6c482875c913.png)

o que devo fazer?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Takepaty, como vai?

No Postman, verifique se você desabilitou a verificação de SSL no menu Configurações.
Se fez isto, verifique se os certificados locais estão OK (Sugir renovar eles):
https://balta.io/blog/dotnet-instalacao-configuracao-e-primeiros-passos#atualizando-os-certificados
